src/iflist.c

```diff
@@ -1138,12 +1138,9 @@ __ni_process_ifinfomsg_linktype(ni_linkinfo_t *link, const char *ifname, ni_netc
 			/* We're at the very least an ethernet. */
 			tmp_link_type = NI_IFTYPE_ETHERNET;
 
-			/*
-			 * Try to detect if this is a  WLAN device.
-			 * The official way of doing this is to check whether
-			 * ioctl(SIOCGIWNAME) succeeds.
-			 */
-			if (__ni_wireless_get_name(ifname, NULL, 0) == 0)
+			/*  Netlink does not tell us if the device has a
```

Comment:
Please use `rtnetlink` unstead of (unspecified)netlink:
```
/*  rtnetlink does not tell us if the device has a
*  wireless extenstions or not, but sysfs does. */
```
Also `extention` is incorrect spelling of `extension`.